tools/twix/src/configuration/keys.rs

```diff
@@ -0,0 +1,233 @@
+use std::{collections::HashMap, fmt};
+
+use eframe::egui::{Key, Modifiers};
+use serde::{
+    de::{self, Deserializer},
+    Deserialize,
+};
+use thiserror::Error;
+
+#[cfg_attr(test, derive(PartialEq))]
+#[derive(Debug, Error)]
+pub enum Error {
+    #[error("Duplicate modifier `{0}`")]
+    DuplicateModifier(String),
+    #[error("Invalid modifier `{0}`")]
+    InvalidModifier(String),
+    #[error("Invalid key `{0}`")]
+    InvalidKey(String),
+}
+
+#[derive(Debug, Deserialize, Clone, Copy, PartialEq)]
+#[serde(rename_all = "snake_case")]
+pub enum KeybindAction {
+    CloseTab,
+    DuplicateTab,
+    FocusAbove,
+    FocusAddress,
+    FocusBelow,
+    FocusLeft,
+    FocusPanel,
+    FocusRight,
+    NoOp,
```

Comment:
What is this for?

Reply:
Unbinding keys defined in the default config. Could be removed if we decide not to merge keybinds with the defaults